crates/witness-generator/src/rpc_generator.rs

```diff
@@ -81,6 +82,8 @@ impl RpcBlocksAndWitnessesBuilder {
 
         Ok(RpcBlocksAndWitnesses {
             client,
+            // TODO: make this dynamic based on the RPC
+            chain_config: ChainConfig::Mainnet,
```

Comment:
We can wrap this in a further PR -- trying to not pack more things in this one.
I can create an issue, but I think we can fix this quickly in a follow up.